src/logic/algo/solver.ts

```diff
@@ -0,0 +1,29 @@
+import { ubcKernel } from "./kernels";
+import { TreeNode } from "../treeBuilder/treeNode";
+
+export interface MonteCarloTreeSearchHyperParams {
+  kernel?: (meanNodeValue: number, visitsOfNode: number, visitsOfParent: number) => number;
+  numRollout: number;
+  numIterations: number;
+}
```

Comment:
@alicjak1519 

Zacząłęm dodawać hiperparametry zgodnie z tym o czym gadaliśmy

Myślę że wszystko oprócz kernela jest dość jasne. Natomiast kernel - to funkcja która dostaje takie trzy parametry jak tam podałem i na podstawie tego uruchamia jakiś powinna zwrócić jakiś skore